library/network/test/network_interfaces_test.rb

```diff
@@ -91,7 +91,7 @@
   describe "#FilterDevices" do
     let(:data_dir) { File.join(File.dirname(__FILE__), "data") }
     # Defined in test/data/etc/sysconfig/ifcfg-*
-    let(:netcard_devices) { ["arc", "bond", "br", "em", "eth", "vlan"] }
+    let(:netcard_devices) { ["arc", "bond", "br", "em", "eth", "ppp", "vlan"] }
```

Comment:
```ppp``` was not handled as netdevice before but modem. So, why was it added there?

Reply:
That's a consequence of 40fca35: we really don't know how an interface may be named, so we basically abolish the name based filtering. So *all* names get through (well, except "lo"). Even ppp. I think the result is less wrong.

Reply:
well that is more less true. But the ```Devices``` cache is organized into "categories" based on device type. This types were then categorized as ("netcard", "modem", ...) and so on. This change means that if you have a device of type ```ppp``` which used to be classified as ```modem``` it will be classified as a ```netcard``` after this update. This might affect e.g. ```yast2 lan``` overview dialog.

Device name is type independent for some time already, so you can have device of ```eth``` type with ```ppp``` name even now.